src/main/java/org/isf/visits/manager/VisitManager.java

```diff
@@ -299,4 +358,9 @@ private String prepareSmsFromVisit(Visit visit) {
 	public Visit findVisit(int id) throws OHServiceException {
 		return ioOperations.findVisit(id);
 	}
+
+	// helper method, TODO could be moved to org.isf.utils.exception.model if the dependency to org.isf.generaldata is ok
+	private static OHExceptionMessage newOHExceptionMessage(String message) {
```

Comment:
Yes, I would move it in `org.isf.utils.exception.model` or I would just add specific constructors for different `OHSeverityLevel`s, wdyt?

Reply:
(yes, dependency is ok)

Reply:
A search for `new OHExceptionMessage(MessageBundle.getMessage("angal.common.error.title")` returns 216 results in the codebase and all of them use `OHSeverityLevel.ERROR`.

I believe a constructor  like `public OHExceptionMessage(String message)` that sets default title to `MessageBundle.getMessage("angal.common.error.title")` and severity to `OHSeverityLevel.ERROR` would be helpful since I think it's natural for an exception to be at the error level by default.

The use of the `WARNING` level when creating `OHExceptionMessage` always sets the title to `null`, ex: 
```java
new OHExceptionMessage(null, "The size should be between 0 and 250!", OHSeverityLevel.WARNING)
```
We could introduce a `public OHExceptionMessage(String message, OHSeverityLevel level)` if you find there is enough value but I would do that in a different PR, PR that could also replace the duplicated the code I mentionned

Reply:
It's ok for another PR, please can you create a Jira issue for that? Thanks!

Reply:
Sure, will do

Reply:
Just to make sure we are on the same page, my understanding is that I will move `newOHExceptionMessage` to a constructor and create a JIRA issue to add `public OHExceptionMessage(String message, OHSeverityLevel level)` and refactor the code to use the `public OHExceptionMessage(String message)` introduced in this PR, correct?

Reply:
yep! for now you can create the issue well explained, then start to work on it after OP-722 is merged. Thanks!

Reply:
JIRA issue created: https://openhospital.atlassian.net/browse/OP-1056